src/_includes/layouts/listing-blog-post-categories.njk

```diff
@@ -38,6 +38,8 @@
 }
 ---
 
+{% include 'components/navigation-secondary.njk' %}
```

Comment:
Pondering: If this is so prevalent, I do wonder whether maybe in future we would create a nested "_subnav" layout that these could all inherit from instead of `base`. Not a big shift, but perhaps just DRYs out the includes throughout. Food for though. Maybe that's an unnecessary layer of misdirection.

Reply:
I'm on the fence with a nested layout.

Let's progress with some page building and see how we feel.